test/e2e-v2/cases/win/mock-data/otel-mock-metrics.json

```diff
@@ -6,7 +6,7 @@
           {
             "key": "service.name",
             "value": {
-              "stringValue": "SkyWalking"
```

Comment:
If a mock e2e test needs a static service name, it is fine.

Reply:
ok.